packages/ffe-radio-button-react/src/RadioSwitch.js

```diff
@@ -0,0 +1,42 @@
+import React, { Fragment } from 'react';
+import { string } from 'prop-types';
+
+import BaseRadioButton from './BaseRadioButton';
+
+const RadioSwitch = (props) => {
+    const {
+        leftLabel,
+        leftValue,
+        rightLabel,
+        rightValue,
+        ...rest
+    } = props;
+
+    return (
+        <Fragment>
+            <BaseRadioButton
+                className="ffe-radio-switch"
```

Comment:
We need the same `classNames` handling here as in `RadioButton`, otherwise a `className` set on the component will override the FFE class.

Reply:
I agree!